libs/ui-toolkit/src/components/toggle/toggle.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Toggle } from './toggle';
+import { useState } from 'react';
+
+export default {
+  component: Toggle,
+  title: 'Toggle',
+} as ComponentMeta<typeof Toggle>;
+
+export const Controlled: ComponentStory<typeof Toggle> = () => {
+  const [checked, setChecked] = useState('test-1');
+
+  return (
+    // Wrapping in a form to scope the effect of the radios (stops light and dark theme versions colliding)
```

Comment:
nitpick (non-blocking): all controls should be in forms anyway

```suggestion
```

Reply:
Aye, it's a good nitpick!